sdk/ai/azure-ai-agents/samples/agents_tools/sample_agents_azure_ai_search.py

```diff
@@ -55,11 +61,6 @@
 )
 
 # Create agent with AI search tool and process agent run
-project_client = AIProjectClient(
-    endpoint=os.environ["PROJECT_ENDPOINT"],
-    credential=DefaultAzureCredential(),
-)
-
 with project_client:
```

Comment:
Unrelated to your change: why doesn't the context start when you create the project_client? I expect to see this in line 50:
 with AIProjectClient(....) as project_client:
     ....